pkg/controller/elasticsearch/nodespec/readiness_probe.go

```diff
@@ -30,7 +30,7 @@ const ReadinessProbeScriptConfigKey = "readiness-probe-script.sh"
 const ReadinessProbeScript string = `
 #!/usr/bin/env bash
 # Consider a node to be healthy if it responds to a simple GET on "/_cat/nodes?local"
-CURL_TIMEOUT=3
+CURL_TIMEOUT=${CURL_TIMEOUT:=3}
```

Comment:
I feel like this env var name should be more explicit: `READINESS_CURL_TIMEOUT`

Reply:
`READINESS_PROBE_TIMEOUT`?

Reply:
Agreed, I'll open a new PR